app/sdk/data/DataSetItemAttachment.java

```diff
@@ -66,22 +67,38 @@ public DataSetItemAttachment() {
         return null;
     }
 
+    public void resizeTo(int width, int height) {
+        Http.MultipartFormData.FilePart oldPart = this.getAttachmentFileItem();
+        File currentFile = (File) this.getAttachmentFileItem().getFile();
```

Comment:
This could cause a NullPointerException. You should check to make sure this actually has a FileItem and File